qp_shotgun/kneaddata/__init__.py

```diff
@@ -0,0 +1,84 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'KneadData', '0.5.1', 'KneadData is a tool designed to perform quality '
+    'control on metagenomic and metatranscriptomic sequencing data, '
+    'especially data from microbiome experiments.')
+
+# Define the HUMAnN2 command
+req_params = {'input': ('artifact', ['per_sample_FASTQ'])}
+opt_params = {
+    # there are other parameters not included that will be ignored in this
+    # configuration as we assume that the correct values were set in the env
+    # by the admin installing the tools:
+    # trimmomatic
+    # bowtie2
+
+    # --input # input FASTQ file (add a second argument instance to run with paired input files)
+    # --output # directory to write output files
+    # --output-prefix # prefix for all output files [ DEFAULT : $SAMPLE_kneaddata ]
+    # --log # filepath for log [ DEFAULT : $OUTPUT_DIR/$SAMPLE_kneaddata.log ]
+    # --trimmomatic # path to trimmomatic executable
+    # --bowtie2 # path to bowtie executable
+    # --bmtagger # path to bmtagger exectuable
+    # --trf # path to TRF executable
+    'reference-db': ['choice:["human_genome"]', 'human_genome'], # ref db
```

Comment:
Here we may want to specify additional reference DBs by including multiple iterations of this flag, one per ref (examples might include PhiX + Human + Mouse). How can we make that work?